docker/.env.default

```diff
@@ -4,6 +4,12 @@ NETWORK=optimism-goerli
 # L1 RPC URL
 L1_RPC_URL=https://eth-goerli.g.alchemy.com/v2/<API_KEY>
 
+# L1 Test RPC URL
+L1_TEST_RPC_URL=https://eth-goerli.g.alchemy.com/v2/a--NIcyeycPntQX42kunxUIVkg6_ekYc
```

Comment:
I don't think we need to include these URLs in the docker configs. The docker setup is not meant to be used to run the tests, but rather for people to run magi as a whole.

Another good reason I want to remove them here is to make sure people aren't tempted to use my keys to sync their node, which will eat up my rate limit quite quickly.

Reply:
Good point, I was surprised you were willing to just leave them in the code, but I guess this makes it even easier for scrapers to use them. Will remove them